server/models/ToDo.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require("mongoose");
+
+const ToDoSchema = new mongoose.Schema(
+  {
+    title: {
+      type: String,
+      required: [true, "Please describe what's to be done."],
+      maxLength: 150,
```

Comment:
Sure, how many would you suggest? 10?

Reply:
I think 2. I would also like to avoid users ability to enter empty strings (i.e. spaces only). I'm not sure if we can check for that type of character within the Schema or if that would be a front end validation check. What do you think?